.github/workflows/gohai.yml

```
@@ -3,6 +3,9 @@ name: "Gohai Test"
 # Only run the tests if pkg/gohai was changed
 on:
   push:
+    branches:
+      - main
+      - 7.[0-9][0-9].x
```

```suggestion
      - 7.[0-9][0-9].x
      - mq-working-branch-*
```
Then I guess we want this to run on MQ branches as well

---

That's what I thought, thanks !